iphone/Maps/LocalizedStrings/de.lproj/Localizable.strings

```diff
@@ -889,6 +889,16 @@
 
 "placepage_bookmark_name_hint" = "Name des Lesezeichens";
 
+/* TODO: translate this */
+"placepage_track_name_hint" = "Track Name";
+
+"placepage_personal_notes_hint" = "Anmerkung";
+
+"placepage_delete_bookmark_button" = "Lesezeichen löschen";
+
+/* TODO: translate this */
+"placepage_delete_track_button" = "Delete Track";
```

Comment:
Please add required strings into data/strings/strings.txt and remove all changes to LocalizedStrings. We will regenerate them properly in upcoming commits.

Reply:
Done